src/components/SurveyQuestion/index.test.tsx

```diff
@@ -6,21 +6,24 @@ import { SurveyQuestion as SurveyQuestionInterface } from 'services/surveyInterf
 import { build } from '@support/factory';
 
 describe('SurveyQuestion', () => {
-  it('renders the given question', async () => {
+  it('renders the given question with order', async () => {
     const question = build('surveyQuestion', {
       displayType: 'rating',
       ratingType: 'star',
+      displayOrder: '2',
     }) as SurveyQuestionInterface;
 
     const { getByText } = render(
       <SurveyQuestion
+        lastQuestionOrder={3}
         question={question}
-        currentResponse={null}
-        setResponse={jest.fn()}
+        currentResponse={undefined}
```

Comment:
Shouldn't we just remove that line? If it is undefined, it is like not setting the prop, isn't it? 🤔

Reply:
(same question applies for similar cases in this PR ;-) )

Reply:
I am trying to replicate the actual value that this component has been used. So in the question page, this component will be passed `undefined` to the `currentResponse` prop. What do you think?

![image](https://user-images.githubusercontent.com/14077479/179717305-9a782584-5cea-4280-bf17-57c8f363d68b.png)

Reply:
Ahhhh, got it! Yes, what you explain makes sense 💯 
Thanks 🙌